cmd/user-cluster-controller-manager/main.go

```diff
@@ -107,6 +110,8 @@ func main() {
 	flag.StringVar(&runOp.clusterURL, "cluster-url", "", "Cluster URL")
 	flag.StringVar(&runOp.dnsClusterIP, "dns-cluster-ip", "", "KubeDNS service IP for the cluster")
 	flag.IntVar(&runOp.openvpnServerPort, "openvpn-server-port", 0, "OpenVPN server port")
+	flag.IntVar(&runOp.kasSecurePort, "kas-secure-port", 6443, "Secure KAS port")
+	flag.Var(&runOp.tunnelingAgentIP, "tunneling-agent-ip", "If specified the tunneling agent will bind to this IP address, otherwise it will not be deployed.")
```

Comment:
Shouldn't we put a default value? We still need a feature-gate flag to enable the new expose strategy right?

Reply:
The value is provided by `seed-controller-manager` when the tunneling strategy is enabled, `192.168.30.10` is the default indeed. We have no feature gate on the `user-controller-manager` at the moment, we are using the `tunnelingAgentIP` presence/absence to decide whether we should deploy the agents or not.